python/lsst/drp/tasks/dcr_assemble_coadd.py

```diff
@@ -1047,6 +1065,48 @@ def calculateConvergence(self, dcrModels, subExposures, bbox, warpRefList, weigh
         self.log.info("Individual metrics:\n%s", metricList)
         return 1.0 if weight == 0.0 else metric / weight
 
+    def calculateTotalConvergence(self, dcrModels, bbox, warpRefList, weightList, statsCtrl):
```

Comment:
Please find a way to re-use `calculateConvergence` with slight modification rather than duplicating the code. For example, you could call `exposure = warpExpRef.get()` if `subExposures` is `None`, and use `exposure = subExposures[visit][bbox]` if it is not `None`.